python/mxnet/space.py

```diff
@@ -1 +1,214 @@
-../../contrib/tvmop/space.py
```

Comment:
?

Reply:
Linux link and Windows shortcut work differently, so I had to paste contrib/tvmop/space.py to python/mxnet/space.py, otherwise python front-end installation fails since Windows recognizes the link file as .py file.

Reply:
Would it be possible to fix the root cause instead? Duplicating the file is not a good strategy as such duplicates tend to get out of sync

Reply:
@hzfan is there any alternative? A workaround I'm using now is to copy the file when running prepare_tvm.ps1 script.

Reply:
I think copying the file at compile time is good. Why not remove this `python/mxnet/space.py` since it will be generated by your workaround? This way we can avoid duplication.

Also, just FYI, if we go with copying, we may have to make some extra modification to ci. Now ci separates the compiling phase and the testing phase. CI will copy the output of compilation (like libmxnet.so) to the testing phase. In our case, `python/mxnet/space.py` becomes part of the compilation output and must be copied.

Reply:
@hzfan If I remove space.py, ONNX build failed probably because it was built before TVM. I think we need a better way of doing this.

Reply:
@vexilligera Could you try adding `python/mxnet/space.py` to `Jenkins_steps.groovy`, so that it will be copied from compilation phase to testing phase